components/inlinealert/index.css

```diff
@@ -13,40 +13,119 @@
 
 .spectrum-InLineAlert {
 	/* Font */
-	--spectrum-inlinealert-heading-font-family: var(--spectrum-sans-font-family-stack);
+	--spectrum-inlinealert-heading-font-family: var(--spectrum-heading-sans-serif-font-family);
 	--spectrum-inlinealert-heading-font-weight: var(--spectrum-heading-sans-serif-font-weight);
 	--spectrum-inlinealert-heading-font-style: var(--spectrum-heading-sans-serif-font-style);
 	--spectrum-inlinealert-heading-font-size: var(--spectrum-heading-size-xxs);
 	--spectrum-inlinealert-heading-line-height: var(--spectrum-heading-line-height);
 
-	--spectrum-inlinealert-content-font-family: var(--spectrum-sans-font-family-stack);
+	--spectrum-inlinealert-content-font-family: var(--spectrum-body-sans-serif-font-family);
```

Comment:
I happened to notice that the heading and body fonts were not loading and were showing up as serif when testing in Assistivlabs. We should be continuing to use the font stack, otherwise Typekit fonts won't load (the font name has a dash). This token has a single value that will only show up when you have the font installed on your local machine. The fonts are a case where we can't match what is listed on the specs.

![Screenshot 2025-05-01 at 11 26 33 AM](https://github.com/user-attachments/assets/fac9d69e-b5a7-483c-a665-bca86ed5af53)